.buildkite/pipeline.yml

```diff
@@ -51,8 +51,44 @@ steps:
           cache-from:
             - node-maze-runner:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:ci-${BUILDKITE_BRANCH}
 
+  - label: ':docker: Build expo maze runner image'
+    plugins:
+      - docker-compose#v2.6.0:
+          build:
+            - expo-maze-runner
+          image-repository: 855461928731.dkr.ecr.us-west-1.amazonaws.com/js
+          cache-from:
+            - expo-maze-runner:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:ci-${BUILDKITE_BRANCH}
+      - docker-compose#v2.6.0:
+          push:
+            - expo-maze-runner:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:ci-${BUILDKITE_BRANCH}
+            - expo-maze-runner:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:ci-latest
+
   - wait
 
+  - label:  ':docker: Publish expo app'
+    env:
+      EXPO_RELEASE_CHANNEL: ${BUILDKITE_BUILD_ID}
+    plugins:
+      - docker-compose#v2.6.0:
+          run: expo-publisher
+          image-repository: 855461928731.dkr.ecr.us-west-1.amazonaws.com/js
+          cache-from:
+            - expo-publisher:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:ci-${BUILDKITE_BRANCH}
+
+  - label:  ':docker: Build expo APK builder'
+    env:
+      EXPO_RELEASE_CHANNEL: ${BUILDKITE_BUILD_ID}
+    artifact_paths: build/output.apk
+    plugins:
+      - docker-compose#v2.6.0:
+          build: expo-android-builder
+          cache-from:
+            - expo-android-builder:855461928731.dkr.ecr.us-west-1.amazonaws.com/js:expo-android-builder-latest
```

Comment:
This should have the branch name as well as a latest cache so it doesnt matter if we have multiple branches going on at the same time.